src/vs/workbench/services/preferences/test/common/keybindingsEditorModel.test.ts

```diff
@@ -199,7 +199,7 @@ suite('Keybindings Editor Model test', () => {
 		});
 	});
 
-	test('convert with title and wihtout binding to entry', () => {
+	test('convert with title and without binding to entry', () => {
```

Comment:
this is probably better to fix upstream in vscode, since this part of the forked source code that we refresh periodically.

Reply:
Thanks @kburtram for the comment, I just did it: https://github.com/Microsoft/vscode/pull/62196

Reply:
Thanks!  I'll leave this PR active waiting to see what they do upstream.  We'd prefer to get the change through the vscode repo if possible.